proto/akash/deployment/v1beta1/deployment.proto

```diff
@@ -47,6 +47,12 @@ message MsgCreateDeployment {
 
   cosmos.base.v1beta1.Coin deposit = 4
       [(gogoproto.nullable) = false, (gogoproto.jsontag) = "deposit", (gogoproto.moretags) = "yaml:\"deposit\""];
+
+  // Depositor pays for the deposit
+  string depositor = 5 [
```

Comment:
Is this going to be storing an account address ?

Reply:
Yes, the address of the depositor's account.